src/api/notify-namespace.ts

```diff
@@ -211,15 +211,15 @@ export class NotifyNamespace {
     let data;
     if ('isActive' in update) {
       restApiName = 'update-webhook';
-      methodName = 'webhooks:updateWebhook';
```

Comment:
perhaps leave the webhooks prefix in the case that non-webhook CRUD fields get added to the CRUD logs?

Reply:
only the SDK sends the header, so it should be fine. I don't foresee a method name collision even if we add non-crud endpoints.